projects/scss-foundation/docs/background-hover.md

```diff
@@ -0,0 +1,17 @@
+# Background hover method
+
+Hover utility for easily define the regular and hover states of background.
```

Comment:
Make this description bit flexible by adding plural from start; in other words:
"Hover utilities for easy definition of regular and hover background states.

Reply:
I noticed that the paths were incorrect in other doc files as well, so I fixed them.

Reply:
That's great unless @Cvetanovic has done it already on other branch :)